bigtable/src/list_instance.php

```diff
@@ -25,30 +25,30 @@
 // Include Google Cloud dependencies using Composer
 require_once __DIR__ . '/../vendor/autoload.php';
 
-if (count($argv) != 3) {
-    return printf("Usage: php %s PROJECT_ID INSTANCE_ID" . PHP_EOL, __FILE__);
-}
-list($_, $projectId, $instanceId) = $argv;
-
 // [START bigtable_list_instances]
-
 use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient;
 
-/** Uncomment and populate these variables in your code */
-// $projectId = 'The Google project ID';
-// $instanceId = 'The Bigtable instance ID';
-
-$instanceAdminClient = new BigtableInstanceAdminClient();
+/**
+ * List Bigtable instances in a project
+ * @param string $projectId The Google Cloud project ID
+ */
+function list_instance($projectId)
```

Comment:
Let's add typehints for all these
```suggestion
function list_instance(string $projectId): void
```